modules/ossm-deploy-application-workloads-per-user-group.adoc

```diff
@@ -0,0 +1,109 @@
+// Module included in the following assemblies:
+// install/ossm-using-multiple-control-planes-on-single-cluster.adoc
+
+:_mod-docs-content-type: Procedure
+[id="ossm-deploy-application-workloads-per-user-group_{context}"]
+= Deploy application workloads per user group
+:context: ossm-deploy-application-workloads-per-user-group
+
+This procedure demonstrates how to deploy application workloads in separate namespaces based on user groups.
+
+.Procedure
+
+. Create an application namespace called `app-ns-1`:
++
+[source,terminal]
+----
+$ oc create namespace app-ns-1
+----
+
+. Label the namespace with the respective control plane:
++
+[source,terminal]
+----
+$ oc label namespace app-ns-1 usergroup=usergroup-1 istio.io/rev=mesh-1
+----
+
+. Deploy the `sleep` and `httpbin` applications:
++
+[source,terminal]
+----
+$ oc apply -f https://raw.githubusercontent.com/openshift-service-mesh/istio/release-1.23/samples/sleep/sleep.yaml
+$ oc apply -f https://raw.githubusercontent.com/openshift-service-mesh/istio/release-1.23/samples/httpbin/httpbin.yaml
+----
+
+. Wait for the `httpbin` and `sleep` pods to run with sidecars injected:
++
+[source,terminal]
+----
+$ oc get pods -n app-ns-1
+----
++
+You should see output similar to the following example:
++
+.Example output
+[source,terminal]
+----
+NAME                       READY   STATUS    RESTARTS   AGE
+httpbin-7f56dc944b-kpw2x   2/2     Running   0          2m26s
+sleep-5577c64d7c-b5wd2     2/2     Running   0          91m
+----
+
+. Repeat the above steps for two more namespaces: `app-ns-2` and `app-ns-3`, adding them both to the `usergroup-2` revision.
```

Comment:
I think we should provide the steps for this explicitly, as the label command is not necessarily that obvious and it's easy to put the wrong thing in the wrong place. I hope it is ok to put multiple commands in some boxes, especially when they are identical and we have already talked about them above. The content would be:

---
1. Create two more namespaces: `app-ns-2` and `app-ns-3`:
```
oc create namespace app-ns-2
oc create namespace app-ns-3
```
2.  Add them both to the second control plane with the discovery selector `usergroup=usergroup-2` and the revision label with the value `mesh-2`:
```
oc label namespace app-ns-2 usergroup=usergroup-2 istio.io/rev=mesh-2
oc label namespace app-ns-3 usergroup=usergroup-2 istio.io/rev=mesh-2
```
3. Deploy the `sleep` and `httbin` applications to both namespaces:
```
oc apply -f https://raw.githubusercontent.com/openshift-service-mesh/istio/release-1.23/samples/sleep/sleep.yaml -n app-ns-2
oc apply -f https://raw.githubusercontent.com/openshift-service-mesh/istio/release-1.23/samples/httpbin/httpbin.yaml -n app-ns-2
oc apply -f https://raw.githubusercontent.com/openshift-service-mesh/istio/release-1.23/samples/sleep/sleep.yaml -n app-ns-3
oc apply -f https://raw.githubusercontent.com/openshift-service-mesh/istio/release-1.23/samples/httpbin/httpbin.yaml -n app-ns-3
```

Reply:
Placing the commands in separate boxes as per doc guidelines.

Reply:
Note: you can convert this into two commands, so you'll only need two boxes instead of four (and the reader will only need to copy-paste 2x instead of 4x):
```
oc apply -n app-ns-2 \
   -f https://raw.githubusercontent.com/openshift-service-mesh/istio/release-1.23/samples/sleep/sleep.yaml \
   -f https://raw.githubusercontent.com/openshift-service-mesh/istio/release-1.23/samples/httpbin/httpbin.yaml

oc apply -n app-ns-3 \
   -f https://raw.githubusercontent.com/openshift-service-mesh/istio/release-1.23/samples/sleep/sleep.yaml \
   -f https://raw.githubusercontent.com/openshift-service-mesh/istio/release-1.23/samples/httpbin/httpbin.yaml
```